modules/virt-hot-plugging-bridge-network-interface-cli.adoc

```diff
@@ -0,0 +1,112 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-hot-plugging-network-interfaces.adoc              
+
+:_content-type: PROCEDURE                                             
+[id="virt-hot-plugging-bridge-network-interface_{context}"]                                   
+= Hot plugging a bridge network interface using the CLI
+
+Hot plug a bridge network interface to a virtual machine (VM) while the VM is running. 
+
+.Prerequisites
+
+* The Multus Container Network Interface (CNI) plugin is deployed on the cluster.
+* A Linux bridge network attachment definition is configured in the same namespace as your VM.
+* You have installed the `virtctl` tool.
+
+.Procedure
+
+. Start the VM to which you want to hot plug the network interface by using the following command:
++
+[source,terminal]
+----
+$ virtctl start <vm_name>
+----
++
+.Example VM configuration
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: vm-fedora
+spec:
+  running: true
+  template:
+    spec:
+      domain:
+        devices:
+          disks:
+          - disk:
+              bus: virtio
+            name: containerdisk
+          interfaces:
+          - masquerade: {}
+            name: defaultnetwork
+          rng: {}
+        resources:
+          requests:
+            memory: 1024M
+      networks:
+      - name: defaultnetwork
+        pod: {}
+      terminationGracePeriodSeconds: 0
+      volumes:
+      - containerDisk:
+          image: quay.io/kubevirt/fedora-with-test-tooling-container-disk:devel
+        name: containerdisk
+----
+
+. Use the following command to hot plug a new network interface to the running VM:
++
+[source,terminal]
+----
+$ virtctl addinterface <vm_name> --network-attachment-definition-name <net_attach_def_name> --name <interface_name>
+----
++
+where:
+
+<vm_name>:: Specifies the name of the `VirtualMachine` object.
+<net_attach_def_name>::  Specifies the name of `NetworkAttachmentDefinition` object.
+<interface_name>:: Specifies the name of the bridge interface.
+
+. Verify that the new interface is added to the VM specification by checking the virtual machine instance (VMI) status:
```

Comment:
This step should come after the migration.
After `virtctl addinterface` the interfaces is only added to the VM/VMI specification not to the status.